src/LanguageFunctions.ts

```diff
@@ -126,6 +126,7 @@ export async function RefreshXlfFilesFromGXlf(sortOnly?: boolean): Promise<{
 }> {
     const xmlns = 'urn:oasis:names:tc:xliff:document:1.2';
     const xmlStub = GetXmlStub();
+    const textNodeValue = '\r\n      ';
```

Comment:
You sure you want do declare with whitespace?

Untested padding function. Not really padding, just adding ;) 
```TypeScript
function pad(str: string, size:number, padString: string = ' '): string {
    let s = "";
    while (s.length < size) s += padString;
    return str + s;
}
```